microprofile/server/src/main/java/io/helidon/microprofile/server/ServerCdiExtension.java

```diff
@@ -113,6 +113,7 @@ private void buildTime(@Observes @BuildTimeStart Object event) {
             throw new IllegalStateException("There is another builder in progress, or another Server running. "
                                                     + "You cannot run more than one in parallel");
         }
+        LOGGER.finest("Builders ready");
```

Comment:
All the log statements for startup logging should be done on 
`private static final Logger STARTUP_LOGGER = Logger.getLogger("io.helidon.microprofile.startup.server");`
not on the usual loggers of these components.

Reply:
This is not "builders ready", but begging of build time processing (the part that is done when building native image or bootstrap of server on JRE).

Reply:
Fixed LOGGER to STARTUP_LOGGER.
Put "builders ready" as in 1.x branch.